platforms/ios/framework/src/TGURLHandler.h

```diff
@@ -0,0 +1,77 @@
+//
+//  TGURLHandler.h
+//  TangramMap
+//
+//  Created by Karim Naaji on 11/23/16.
+//  Updated by Matt Blair on 7/16/18.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
```

Comment:
Missing `#pragma once`

Reply:
I think we _could_ add `#pragma once` and nothing bad would happen, but so far we've consistently not used it in Objective-C headers because we always `#import` them - which makes sure they're only included once.